modules/ols-about-data-telemetry-transcript-and-feedback-collection.adoc

```diff
@@ -0,0 +1,12 @@
+// This module is used in the following assemblies:
+// configure/ols-using-openshift-lightspeed.adoc
```

Comment:
// about/ols-about-openshift-lightspeed.adoc ?

Reply:
yes, @syedriko  thank you for noticing this. I copied the header from a different file and forgot to change this.

Reply:
done